### PR TITLE
Lesson 1.8: Data Fetching/ Data Refetching/ Memoized Handlers

### PR DESCRIPTION
**Pull request by nihalerdal**
Here is my lesson_1_8 pull request for the review.

### CODE REVIEW

File: src/AddTodoForm.jsx [hunk 1]
```diff
@@ -11,9 +11,8 @@ function AddTodoForm({ onAddTodo }) {
   };
 
   const handleAddTodo = (event) => {
-    event.preventDefault();
```
Comment (AnnaPestova1) [inline src/AddTodoForm.jsx:14]: because you removed event.preventDefault you cannot add new todo

**Review by AnnaPestova1 — CHANGES_REQUESTED** [review]
Great job! fix according the comments, please

File: src/App.jsx [hunk 2]
```diff
@@ -4,50 +4,109 @@ import AddTodoForm from "./AddTodoForm"; //Import AddTodoForm component
...
+    try {
+      const response = await fetch(url, options);
+      
+      if (!response.ok) {
+        throw new Error(`Error: ${response.status}`);
+      }
+
+      const data = await response.json();
+  
+      const newTodo = {
```
Comment (AnnaPestova1) [inline src/App.jsx:82]: try to console.log data, you have some issue in the newTodo

File: src/AddTodoForm.jsx (hunk 1 above)
Comment (nihalerdal) [reply src/AddTodoForm.jsx:14]: Fixed! Here is the commit: https://github.com/nihalerdal/react-todo/commit/f94100320e9b20c753b4226f12a04f19fe80b362

File: src/App.jsx (hunk 2 above)
Comment (nihalerdal) [reply src/App.jsx:82]: Resolved! https://github.com/nihalerdal/react-todo/commit/f94100320e9b20c753b4226f12a04f19fe80b362

**Review by AnnaPestova1 — APPROVED** [review]
Hi Nihal! Thank you for changes. Everything looks great!

**Pull request merged**